views/base.njk

```diff
@@ -136,5 +136,82 @@
 
     {% block main_content %}
     {% endblock %}
+
+    <footer class="usa-footer usa-footer-medium" role="contentinfo">
+        <div class="usa-footer-primary-section">
```

Comment:
minor: we have 2-space indentation throughout the rest of this file. Could you make these section also adhere to that?